cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/health_check/health_check_manager.conf.erb

```diff
@@ -0,0 +1,2 @@
+[health_check_manager]
+managed_health_check_dir = <%= node['cluster']['slurm']['install_dir'] %>/etc/pcluster/.slurm_plugin/scripts/health_checks
```

Comment:
From the log you added in the PR description the gpu check is expensive (about 1 minute), I would expect to have a parameter in the configuration of the daemon to disable it.